.jenkins/pytorch/win-test-helpers/test_python_all_except_nn.bat

```diff
@@ -1,3 +1,3 @@
 call %SCRIPT_HELPERS_DIR%\setup_pytorch_env.bat
-cd test && python run_test.py --exclude-jit-executor --verbose --determine-from="%1" && cd ..
+cd test && python run_test.py --exclude-jit-executor --exclude test_nn --verbose --determine-from="%1" && cd ..
```

Comment:
LOL. Someone else removed it 6 months ago in https://github.com/pytorch/pytorch/commit/edc5ef1afb585005408ef8c64ea4fa571ae4c3a0 and no one have noticed that.

Reply:
The file is named `test_python_all_except_nn.bat` but we forget to exclude `test_nn`.

Reply:
Yes I saw that... Given the commit you linked, this was most likely removed by mistake when trying to exclude some jit stuff...
cc @Krovatkin @suo who wrote and reviewed the corresponding PR in case there is something more and this was done on purpose?

Reply:
Is there a reason why we are skipping test_nn on Windows? (if there is an issue tracking that, can you please reference it in the comment)

Reply:
@malfet We split the Windows test into two. The first job already runs the nn-related tests, so we skip them in the second one here.

Reply:
The work is continued in https://github.com/pytorch/pytorch/pull/50660.